### ISSUE TITLE
agent_control should allow you to unblock an IP

### ISSUE THREAD

**Issue by mitchya1** [wazuh/wazuh#1491]
## Feature request

**Compatibility**
Manager

**Component involved**
Active Response

Description
--------------
The agent_control command should take an argument to unblock an IP address. Analysisd occasionally registers some false positives around SQL injections. 

Example: `A%20rider%20from%20the%20University%20of%20California-Davis%20during%20the%20womens%20DI%20criterium.jpg?m=1304897547`

This is a non-malicious request for an image with spaces in the filename.  Analysisd registered this as an attempted SQL injection and blocked the IP. It'd be nice to have an easy way to unblock IPs without having to wait for the timeout to expire. 

If there _is_ a way to unblock an IP using agent_control (or a different command), I'd love to know!

**Comment by vikman90** [comment]
Hi @mitchya1,

This is a good point. We are gradually moving the agent-server interaction to the API, so I think this feature would add it much value.

We have currently an API call (`PUT`) to apply an active response. Maybe we can implement a `DELETE` call to stop it (e.g. unblock an IP).

Pinging @mgmacias95 to add this feature to our roadmap.

Thank you very much for your feedback!
Best regards.

**Comment by oliveiralive** [comment]
Hi guys, this feature already exist?

**Comment by oliveiralive** [comment]
Hi Team, i found a way to unblock IP by wazuh manager

First you need to create 4 AR in manager, 
2 whithout time-out, this will be put um production
2 whith time-out of 10 seconds


For Windows
![image](https://user-images.githubusercontent.com/35010493/187787488-127336e0-742b-4029-8e80-a601d25d1418.png)



For Linux
![image](https://user-images.githubusercontent.com/35010493/187787520-f67d54de-5fdb-4a2d-8c02-499f5448d5e8.png)
 

When same attack happening in your network the AR will block the IP whitout timeout
Then you can unblock the IP by manager whit the agent_control like below
![image](https://user-images.githubusercontent.com/35010493/187787751-677adc99-0ca3-4f20-bce5-b142628fe6d2.png)
